tests/test_parse.py

```diff
@@ -243,3 +243,77 @@ def test_parse_encoding():
         jsxml = js2xml.parse(snippet, encoding=encoding)
         result = jsxml.xpath("//string/text()")
         assert_equal(result, expected)
+
+
+def test_keywords_as_object_keys():
+    # https://www.w3schools.com/js/js_reserved.asp
```

Comment:
```suggestion
    # https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords
```
Not a deal-breaker, but I think linking to MDN might be better.